src/uwtools/resources/schema_with_metatasks.rng

```diff
@@ -257,12 +257,6 @@
         <empty />
       </element>
       <ref name="rb_or_sh" />
-      <element name="true">
-        <empty />
-      </element>
-      <element name="false">
-        <empty />
-      </element>
```

Comment:
Removed `true`/`false` tags from the schema. They already did not exist in the JSONSchema.

Reply:
Apart from formatting, I don't think we'd want to change the RELAX NG schema: It should still be able to validate a Rocoto document, even if it's not one that we produce. I don't think it should do any harm to leave the true/false entities in the RELAX NG schema; we just don't want them in the JSON Schema.

Reply:
Sounds good, undoing these changes very quick